src/components/Header.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect } from 'react'
+import { Link } from 'react-router-dom'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faHome, faUserCircle, faBell } from '@fortawesome/free-solid-svg-icons'
+import Search from './Search'
+import { useGlobalContext } from '../store/global'
+
+export default function Header() {
+
+  const { user, getUser } = useGlobalContext(user)
+
+  // if user has logged in before and we have their token and id, log them in
+  useEffect(() => {
+    if (!user) {
+      const id = localStorage.getItem('user')
+      const token = localStorage.getItem('token')
+      if (id && token) {
+        getUser(id, token)
+      }
+    }
+  }, [])
+
+	return (
+		<div className="header">
```

Comment:
I've noticed some weird formatting. Might want to run `prettier` on the codebase.

Reply:
Done!